packages/connectors/connector-oauth2/package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@logto/connector-oauth",
-  "version": "1.4.0",
+  "version": "1.5.0",
```

```suggestion
  "version": "1.4.0",
```
There is no need to manually bump the connector version since we are using "[changeset](https://github.com/changesets/changesets)" to do the version control.